SDK_versioned_docs/version-3.0.0/guides/auto-router/swap-and-add.md

```diff
@@ -0,0 +1,147 @@
+---
+id: swap-and-add
+title: Swap and Add Liquidity Atomically
+sidebar_position: 2
+---
+
+# Swap and Add Liquidity Atomically
+
+You can use the Auto Router to fetch optimized trade routes that result in token balances proportional to the amount needed for adding optimal liquidity to a pool position. Use the method `routeToRatio` to get quotes, gas information, and calldata for atomically swapping tokens to a particular ratio and adding the new token balances to a pool position.
+
+## Initializing the Alpha Router
+
+Follow the instructions to [import](https://docs.uniswap.org/sdk/guides/auto-router/quick-start#importing-the-package) then [initialize](https://docs.uniswap.org/sdk/guides/auto-router/quick-start#initializing-the-alpharouter) an instance of the Alpha Router
```

Comment:
```suggestion
First, [import](https://docs.uniswap.org/sdk/guides/auto-router/quick-start#importing-the-package) and [initialize](https://docs.uniswap.org/sdk/guides/auto-router/quick-start#initializing-the-alpharouter) an instance of the Alpha Router. If you are using a different network, be sure to specify the right `chainId` parameter.

<code>
import { AlphaRouter } from "@uniswap/smart-order-router";
const router = new AlphaRouter({ chainId: 1, provider: web3Provider };
</code>

```

Reply:
let's include the complete code example to make this as easy as possible for newcomers

Reply:
github UI is not letting me wrap those two lines in code. Please do before merging!

Reply:
also, should we add a command line in a separate code file (since its for the terminal) `> npm install @uniswap/smart-order-router`